int32 hours
int32 minutes
int32 seconds
int32 total_people
int32 people_found
int32 people_missing